static/app/views/issueList/customViewsHeader.tsx

```diff
@@ -48,12 +49,12 @@ function CustomViewsIssueListHeader({
     orgSlug: props.organization.slug,
   });
 
+  const [headerBorderStyle, setHeaderBorderStyle] = useState<'dashed' | 'solid'>('solid');
```

Comment:
It seems like a better way of fixing this would be to update the `useEffect` in this file that reacts to the missing viewId/query/sort. Right now it only runs on first render because of the empty deps array, but it should probably run whenever the URL is updated. 

I think I'd rather not add this state now. It adds to the complexity and might make it harder to find the correct solution later

Reply:
Good idea, didn't think of that!